src/reactive-handler.ts

```diff
@@ -145,16 +143,19 @@ export class ReactiveProxyHandler {
             return desc;
         }
         const shadowDescriptor = getOwnPropertyDescriptor(shadowTarget, key);
-        if (!desc.configurable && !shadowDescriptor) {
+        if (!isUndefined(shadowDescriptor)) {
+            return shadowDescriptor;
```

Comment:
being in the shadowTarget means that there is nothing else to do here, we can reliable use that one.

Reply:
I don't understand this comment.

Reply:
the `shadowTarget` is use to preserve the object invariants when the originalTarget has a non-configurable descriptor. In this condition, we check if we already have the descriptor fro the current key, and if we do, that's what we should return.